input/docs/png_vs_jpg.md

```diff
@@ -0,0 +1,90 @@
+title: PNG vs JPG
+description: Comparing .png and .jpg formats
+---
+>This document works with MATLAB built-in image *coloredChips.png* in `A` variable. The code uses `A` to create images in an **imgs/** folder.
+# Creating an image database
+* to compare atributes of both formats, following code will create database of 100 **.jpg** and 100 **.png** images of *coloredChips.png*, varying in size. 
+* First create **imgs/** folder in your actual working directory
+``` matlab
+clear all 
+
+route= 'imgs/'; %sets route to saving directory
+A=imread('coloredChips.png'); %reads an image
+
+for i=1:100
+	pic_res=imresize(A,i/100); %img resize
+	[r, c]=size(pic_res);
+	pic_size(i,1)=r*c; %size of current img in pixels
+
+	imwrite(pic_res,[route ,'pic_' num2str(i/100) '.jpg'],'JPEG'); %saving current img in .png
+	imwrite(pic_res,[route, 'pic_' num2str(i/100) '.png'],'PNG'); %saving current img in .jpg
+end
+
+```
+#Loading of .jpg images
+``` matlab
+B = dir([route,'*.jpg']); % load all .jpg images into B 
+B_count= length(B); % number of .jpg images
+index = 1;
+```
+# Counting pixels and bytes of each .jpg image
+``` matlab
+for img_Index = 1: B_count
+	act_img = B(img_Index);
+	
+	if(~act_img.isdir)
```

Comment:
Myslím, že to tu být nemusí. Jelikož do adresář být nemůže, pokud jsme to vyselektovali na soubory s .jpg příponou.. Kód v souboru skutečně není dokonalý a chce se to nad tím zamyslet a vyzkoušet...